.github/workflows/image.yaml

```diff
@@ -70,6 +70,7 @@ jobs:
         env:
           IMAGE_NAME: ghcr.io/${LOWERCASE_REPO_OWNER}/k8s-device-plugin
           VERSION: ${COMMIT_SHORT_SHA}
+          IMAGE_EXTRA_TAG_NAMES: ${{ github.base_ref }}
```

Comment:
Why is the SHA-tagged image not sufficient?

Reply:
For nightly tests we won't have a SHA, Nightly test will be run against a branch tag (main)